### PR TITLE
Bump all go1.12 versions to 1.12.12

### PR DESCRIPTION
**Pull request by cblecker**

### CODE REVIEW

File: images/krte/variants.yaml [hunk 1]
```diff
@@ -3,30 +3,30 @@
...
+    GO_VERSION: 1.12.12
     K8S_RELEASE: stable-1.15
     BAZEL_VERSION: 0.23.2
   '1.14':
     CONFIG: '1.14'
-    GO_VERSION: 1.12.10
+    GO_VERSION: 1.12.12
     K8S_RELEASE: stable-1.14
-    BAZEL_VERSION: 0.21.0
+    BAZEL_VERSION: 0.23.2
```
Comment (krzyzacy) [inline images/krte/variants.yaml]: what's the rational to bump bazel for an older release here? I thought that should not change?

Comment (cblecker) [reply images/krte/variants.yaml]: This was needed to support a rules_go bump that was also needed to bump Go. All the go 1.12 versions use the same bazel version, and the rules_go they use is all compatible.

**Pull request merged**